include/LightGBM/config.h

```diff
@@ -364,7 +364,8 @@ struct Config {
   // alias = fs, forced_splits_filename, forced_splits_file, forced_splits
   // desc = path to a ``.json`` file that specifies splits to force at the top of every decision tree before best-first learning commences
   // desc = ``.json`` file can be arbitrarily nested, and each split contains ``feature``, ``threshold`` fields, as well as ``left`` and ``right`` fields representing subsplits
-  // desc = categorical splits are forced in a one-hot fashion, with ``left`` representing the split containing the feature value and ``right`` representing other values
+  // desc = categorical splits are forced in a one-hot fashion, with ``left`` representing the split containing the feature value and ``right`` representing other values.
+  // desc = Note: the forced splic logic will be ignored if the split make gain worse.
```

Comment:
Typos: `splic` -> `split`, `make` -> `makes`.

Also, can you please remove dot at the end and make the word "Note" bold (`**Note**:`) for the consistency with other descriptions?